### ISSUE TITLE
Backup bucket name really optional?

### ISSUE THREAD

**Issue by jrgarner** [plus3it/cfn-collibra#19]
### Problem Description:

Seems a little odd that the "backup bucket" name would be optional. If so, what name is created/used?

### Expected Behavior:

Name is a "required" value

### Actual Behavior:

Name is tagged as "optional".

### (Detailed) Steps to reproduce:

Run the 01_Create_S3-backup_bucket pipeline

### (Optional) Fix recommendation:

Can we make mandatory?

**Comment by ferricoxide** [comment]
Re:
> Seems a little odd that the "backup bucket" name would be optional.

S3 bucket names _must_ be globally unique. It  actually makes life a _ton_ easier to just let AWS name the bucket.

> If so, what name is created/used?

After a stack is deployed from the [S3 CFn template](https://github.com/plus3it/cfn-collibra/blob/master/Templates/make_collibra_S3-bucket.tmplt.json), you can look under the stack's `Outputs` tab for the `CollibraBucketName` item. It will contain the name of the S3 bucket that the template produced.

In short, this behavior is as designed and will be a `WONTFIX`.

**Comment by jrgarner** [comment]
That's cool just need to document it.
